src/annmas/scsplit/command.py

```diff
@@ -0,0 +1,300 @@
+import logging
+import sys
+import itertools
+
+import time
+
+import click
+import click_log
+import tqdm
+
+import pysam
+import multiprocessing as mp
+
+from inspect import getframeinfo, currentframe, getdoc
+
+from ..utils.model import array_element_structure
+
+from ..annotate.command import SegmentInfo
+from ..annotate.command import SEGMENTS_TAG
+from ..annotate.command import _get_segments
+
+from ..utils.constants import __SENTINEL_VALUE
+
+from ..meta import VERSION
+
+logging.basicConfig(stream=sys.stderr)
+logger = logging.getLogger("scsplit")
+click_log.basic_config(logger)
+
+
+__DEFAULT_DUMMY_CBC = "CTGCCTAACCTGATCC"
+__DEFAULT_OUT_BASE_NAME = "scsplit"
+__DEFAULT_UMI_LENGTH = 10
```

Comment:
Is the UMI 10-bp long or 11-bp in these datasets?  At the current setting, it looks like it's taking 11 bp:

@m64020_201213_022403/29/ccs/0_578/10x_Adapter-3p_Adapter
CTGCCTAACCTGATCC**GTAGTCAGGGG**
+
]]]]]]]]]]]]]]]]ebgW/]R3o}{

Reply:
Oops.  Definitely supposed to be 10 bp.  Looks like a mistake with inclusive / 0-based coordinates.